### PR TITLE
Update action.yml

### PR DESCRIPTION
**Pull request by tirthpatel7498**
Guys, kindly check if anything else is required to be added to `index.js` and `action.yml` files. If so, kindly suggest so that I can add those things.

### CODE REVIEW

File: action.yml [hunk 1]
```diff
@@ -1,8 +1,12 @@
-name: 'Wait'
+name: 'Constable'
 description: 'Create a JavaScript Action grade the project'
+inputs:
+
 outputs:
-  grade:
-    description: 'The output of this file will be a grade and percentage given to the repo.'
+  score:
```
Comment (dangoslen) [inline action.yml:6]: I thought we wanted `grade`, `score`, and `report` based on our issue conversation. 

https://github.com/dangoslen/constable-github-action/issues/4

File: action.yml [hunk 2]
```diff
@@ -1,8 +1,12 @@
...
+    description: 'The repository is assigned a score based on the evaluation criterias'
+  feedback:
+    description: 'Feedback on how to improve the repo is provided'
 runs:
   using: 'node12'
-  main: 'dist/index.js'
+  main: 'main/index.js'
```
Comment (dangoslen) [inline action.yml]: ? where is the `main` directory?

We should be packaging the index.js file via ncc. The command for that is `npm run package` and then GitHub uses that file for running. So this should stay as `dist/index.js`

**Comment by dangoslen** [comment]
@tirthpatel7498 

I don't see any updated tests - do you mind adding a simple one to make sure we calculate the right score/grade based on the presence of the files?

File: action.yml (hunk 2 above)
Comment (tirthpatel7498) [reply action.yml]: My bad. main folder came with the action.yml file before I started editing it.

File: action.yml (hunk 1 above)
Comment (tirthpatel7498) [reply action.yml:6]: So I am keeping the following outputs:
1. Score
2. Grade
3. Report
Hope this is what you guys are expecting.

**Comment by tirthpatel7498** [comment]
> @tirthpatel7498
> 
> I don't see any updated tests - do you mind adding a simple one to make sure we calculate the right score/grade based on the presence of the files?

Yes, I'll add them. Thanks

**Comment by dangoslen** [comment]
The test that runs the action is still failing btw. Looks like an issue with the `action.yaml`

https://github.com/dangoslen/constable-github-action/pull/14/checks?check_run_id=1093162226

maybe we should add a `.editorconfig` for `.yaml` - https://editorconfig.org/

**Pull request closed without merging**